config/swagger.yaml

```diff
@@ -54,9 +54,8 @@ paths:
       responses:
         '200':
           description: 'Successful operation'
-          oneOf:
-            - $ref: '#/definitions/KeyBlock'
-            - $ref: '#/definitions/MicroBlockHeader'
+          schema:
+            $ref: '#/definitions/KeyBlockOrMicroBlockHeader'
```

Comment:
I see this introduced an extra level of property key_block/microblock. Are:
* System tests
* User doc (installation, configuration, operation) in release notes
* User doc in protocol repo

up to date?

Reply:
System tests yes, I'm going through the rest.